serverless/tensorflow/mediapipe/nuclio/main.py

```diff
@@ -0,0 +1,80 @@
+import mediapipe as mp
+from mediapipe.tasks import python
+from mediapipe.tasks.python import vision
+import json
+import base64
+import io
+from PIL import Image
+import yaml
+import numpy as np
+def init_context(context):
+    context.logger.info("Init detector...")
+
+    base_options = python.BaseOptions(
+    model_asset_path='pose_landmarker_heavy.task'
+    )
+    options = vision.PoseLandmarkerOptions(
+        base_options=base_options,
+        num_poses = 2,
+        min_pose_detection_confidence = 0.5,
+        min_pose_presence_confidence = 0.5,
```

Comment:
Shall we just initialize them by thershold and remove ``if`` conditions at the end of the file?